src/test_encode_decode_aom.rs

```diff
@@ -207,6 +222,18 @@ fn reordering() {
   }
 }
 
+#[test]
+fn issue_890() {
```

Comment:
Might be nicer to have a self-descriptive name and a comment referring to the issue, as @rzumer pointed out